modules/vstudio/vs2010_vcxproj.lua

```diff
@@ -744,7 +744,7 @@
 ---
 	m.categories.ClCompile = {
 		name       = "ClCompile",
-		extensions = { ".cc", ".cpp", ".cxx", ".c", ".s", ".m", ".mm" },
+		extensions = { ".cc", ".cpp", ".cxx", ".c", ".s", ".m", ".mm", ".ixx" },
```

Comment:
I'm not overly keen on this change. `.ixx` can be used for inline definitions which should be included by another file, rather than compiled themselves. VS2017 recognizes (and seems to use) the extension for C++ Modules but this isn't part of any standard and I can't really see anything that indicates that it needs to be explicitly compiled? I think this is better suited as a local override, using [`buildaction'](https://github.com/premake/premake-core/wiki/buildaction) like this:
```lua
filter { "files:**.ixx" }
  buildaction "ClCompile"
filter {}
```